README.md

```diff
@@ -67,7 +67,7 @@ curl http://localhost:9003/writetagged -s -X POST -d '{
 
 ```
 curl http://localhost:9003/query -s -X POST -d '{
-  "namespace": "metrics",
+  "namespace": "default",
```

Comment:
Why is this changed?

Reply:
The default namespace is `default` so just makes sense to have the docs reflect that:
```
var (
	defaultLocalConfiguration = &config.LocalConfiguration{
		Namespace: "default",
		Retention: 2 * 24 * time.Hour,
	}
)
```